src/aro/target.zig

```diff
@@ -685,9 +641,8 @@ pub fn toLLVMTriple(target: std.Target, buf: []u8) []const u8 {
         .dragonfly => "dragonfly",
         .freebsd => "freebsd",
         .fuchsia => "fuchsia",
-        .kfreebsd => "kfreebsd",
         .linux => "linux",
-        .lv2 => "lv2",
+        .ps3 => "ps3",
```

Comment:
The LLVM name shouldn't be changed.

Reply:
Why so? It has been changed here https://github.com/ziglang/zig/pull/20708/files#diff-73b15ac9aed4b9aca85583d8caa883b2a5211a63b411ae6ef088a6d8b92156f6L671

Reply:
not sure what did you mean

Reply:
Oh, I'm sorry. Done